src/resolvers/user_management.js

```diff
@@ -25,27 +25,54 @@ export default {
 
       return true
     },
-    login: async (parent, { email, password }, { driver, req, user }) => {
+    login: async (_, { email, password }, { driver, req, user }) => {
       // if (user && user.id) {
       //   throw new Error('Already logged in.')
       // }
       const session = driver.session()
-      return session.run(
-        'MATCH (user:User {email: $userEmail}) ' +
-        'RETURN user {.id, .slug, .name, .avatar, .email, .password, .role} as user LIMIT 1', {
-          userEmail: email
-        })
-        .then(async (result) => {
+      return session
+        .run(
+          "MATCH (user:User {email: $userEmail}) " +
+            "RETURN user {.id, .slug, .name, .avatar, .email, .password, .role} as user LIMIT 1",
+          {
+            userEmail: email
+          }
+        )
+        .then(async result => {
           session.close()
-          const [currentUser] = await result.records.map(function (record) {
-            return record.get('user')
+          const [currentUser] = await result.records.map(function(record) {
+            return record.get("user")
```

Comment:
Please use single quotes where possible.

Reply:
sorry, will configure my prettier to use single quotes.